.github/workflows/backend.yml

```diff
@@ -174,7 +174,7 @@ jobs:
           extensions: ${{ env.extensions }}
           coverage: none
           tools: |
-            composer:2.3.7, composer-normalize:2.28.0, composer-require-checker:4.0.0,
+            composer:2.4.1, composer-normalize:2.28.3, composer-require-checker:4.1.0,
```

Comment:
I've done few tests. `composer-normalize` doesn't support composer versions newer than 2.3.8.

Actually it seems to support 2.3.9 but I guess there is another problem.
https://github.com/ergebnis/composer-normalize/pull/942

Reply:
thanks! downgraded for now